.github/workflows/release.yml

```diff
@@ -83,9 +83,9 @@ jobs:
           GITHUB_TOKEN: ${{ steps.generate_token.outputs.token }}
         run: |
           echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" >> .npmrc
-          npx nx release --skipPublish
-          git push --follow-tags
-          npx nx release publish
+          npx nx release --yes
+      - name: Collect and upload Code PushUp report
+        run: npx nx code-pushup -- autorun
```

Comment:
This is now being run by the subsequent `code-pushup/github-action` step.

```suggestion
```

Reply:
✔️